### PR TITLE
Update profile header styles

### PR DESCRIPTION
**Pull request by sanlouise**
[TICKET](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9059)

## Description
We want to remove 'Your profile' from both mobile and desktop, and remove the badge on mobile. On Desktop we decided to make the badge smaller, 75px.

We add the h1 to the 'Your Profile' text in the side nav.

## Testing done
Works locally

## Screenshots
image
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by erikphansen** [comment]
On desktop should the header be so far to the left? Previously the text was left aligned right over the "Personal Information" section header

**Comment by sanlouise** [comment]
> On desktop should the header be so far to the left? Previously the text was left aligned right over the "Personal Information" section header

@erikphansen Good eye! But the change is intended and discussed with Tressa :)

**Comment by erikphansen** [comment]
@sanlouise ok. bummer this looks a little odd to me now. oh well! not my call! :)

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx [hunk 1]
```diff
@@ -93,11 +90,9 @@ const ProfileHeader = ({
...
     ].join(' '),
-    serviceBadge: [
-      ...serviceBadgeClasses,
-      ...serviceBadgeClassesMedium,
-      'usa-width-one-fourth',
-    ].join(' '),
+    serviceBadge: [...serviceBadgeClasses, ...serviceBadgeClassesMedium].join(
+      ' ',
+    ),
     title: [...titleClasses, ...titleClassesMedium].join(' '),
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/ProfileHeader.jsx:96]: can we get rid of this now?

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx [hunk 2]
```diff
@@ -39,12 +39,9 @@ const ProfileHeader = ({
...
-  const serviceBadgeClasses = prefixUtilityClasses([
-    'text-align--center',
-    'margin-bottom--2',
-  ]);
+  const serviceBadgeClasses = prefixUtilityClasses(['display--none']);
   const serviceBadgeClassesMedium = prefixUtilityClasses(
-    ['text-align--right', 'margin-bottom--0', 'padding-right--2'],
+    ['display--flex', 'justify-content--flex-end', 'margin-bottom--0'],
     'medium',
   );
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/ProfileHeader.jsx:47]: can we get rid of the `titleClasses` now?

File: src/applications/personalization/profile-2/components/ProfileHeader.jsx [hunk 3]
```diff
@@ -113,14 +108,13 @@ const ProfileHeader = ({
...
               alt={`${latestBranchOfService} seal`}
-              className="profile-service-badge"
+              className="profile-service-badge vads-u-padding-right--3"
               aria-hidden="true"
               role="presentation"
             />
           )}
         </div>
         <div className="name-and-title-wrapper">
-          <h1 className={classes.title}>Your Profile</h1>
```
Comment (erikphansen) [inline src/applications/personalization/profile-2/components/ProfileHeader.jsx:123]: 😱 no `h1` on the page anymore! Do we have a ticket to redo the sidenav or are we waiting on Tressa to finish up mobile nav design changes first?

Comment (sanlouise) [reply src/applications/personalization/profile-2/components/ProfileHeader.jsx:123]: Just messaged Tressa to confirm, but I will wait with merging this ticket till this is sorted out

**Review by erikphansen — APPROVED** [review]
I meant to approve this yesterday

**Review by erikphansen — APPROVED** [review]

**Pull request merged**